google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -249,6 +263,7 @@ public void terminated(State from) {}
   private static class SubscriberImpl extends AbstractService {
     private static final Logger logger = LoggerFactory.getLogger(Subscriber.class);
 
+    private final SubscriptionName subscriptionName;
     private final String subscription;
```

Comment:
Maybe rename the String to "cachedSubscriptionNameString" to indicate that the authoritative data source is subscriptionName.